DataFormats/TriggerDetId/BuildFile.xml

```diff
@@ -0,0 +1,6 @@
+<use   name="FWCore/Utilities"/>
```

Comment:
Given the previous comment, there is no need for TriggerDetId package and HGCTriggerDetId should be part of the ForwardDetId package. 

This is giving some foresight to including it in CMSSW for real.